tiatoolbox/data/remote_samples.yaml

```diff
@@ -16,6 +16,9 @@ files:
   dicom-1:
     url: [*wsis, "CMU-1.dicom.zip"]
     extract: True
+  ngff-1:
```

Comment:
Can we also take a sample from links provided by Glencoe?

Reply:
I don't think they had samples for v0.4 last time I checked but I will try to check again

Reply:
The latest brightfield sample I have is on v0.1. I believe that Glencoe mentioned that there should be some out there last time we spoke. I'll check emails etc to see if I can find a source of version 0.4 images.

Reply:
The latest update on NGFF was in December in this [blog post](https://www.openmicroscopy.org/2021/12/16/ome-ngff.html). @John-P there are some samples that are of version 0.3 at the end of the post.